lib/multicell_location/services/nrf_cloud_integration.c

```diff
@@ -76,8 +76,11 @@ static void location_service_location_ready_cb(const struct nrf_cloud_cell_pos_r
 		nrf_cloud_location.latitude = result->lat;
 		nrf_cloud_location.longitude = result->lon;
 		nrf_cloud_location.accuracy = (double)result->unc;
+
+		k_sem_give(&location_ready);
+	} else {
+		k_sem_reset(&location_ready);
 	}
-	k_sem_give(&location_ready);
```

Comment:
please keep k_sem_give() after the `if` statment only.

Reply:
I am not sure I understand what you mean. The point of this PR was to reset the semaphore if result == NULL in order to  return -ETIMEDOUT at line 113. I could actually add a comment here to make it clear why we are doing this.

Reply:
Oh! excuse me my ommition - was reviewing too sketchy and mistook the calls as the same